src/config.js

```diff
@@ -0,0 +1,11 @@
+//const env = process.env.NODE_ENV; // 'dev' or 'test'
+
+const config = {
+  host: `http://localhost:8081`,
+  port: 8081,
+  AppID: `7c0c8e717c744b103b263ef8b89d17cc429e6cfab0cddb8b3df21b66e21b45c1`,
+  APP_SECRET: `${process.env.APP_SECRET}`,
+  redirect_url: `http://localhost:8081/redirect`,
```

Comment:
You are repeating yourself here. The redirect_url is based on the `host`, you will need to update twice if the host is updated.